grpc-interface-support/src/main.py

```diff
@@ -54,34 +122,58 @@ def create_service_sdk_dir(proto_file_handle: proto.ProtoFileHandle) -> str:
     return service_sdk_path
 
 
-def generate_single_service(
+def generate_services(
     factory: GrpcServiceSdkGeneratorFactory, if_config: Dict[str, Any]
 ) -> None:
-    """Generate an SDK for a single service.
+    """Generate an SDK for service(s).
 
     Args:
         factory (GrpcPackageGeneratorFactory):
             The factory from which to generate an SDK generator for a single service.
         if_config (Dict[str, Any]): The grpc-interface config.
     """
-    print(
-        f"Generating service SDK for {create_truncated_string(if_config['src'], 100)!r}"
-    )
-    proto_file_handle = proto.ProtoFileHandle(obtain_local_file_path(if_config["src"]))
-    service_sdk_dir = create_service_sdk_dir(proto_file_handle)
 
+    proto_file_handles = fetch_protos(obtain_local_file_path(if_config["src"]))
     is_client = "required" in if_config
     is_server = "provided" in if_config
 
+    for _proto in proto_file_handles:
```

Comment:
```suggestion
    for proto in proto_file_handles:
```

Why is this local variable prefixed with an underscore :D?

Reply:
else name shadowing but I can rename

Reply:
maybe call it `proto_file_handle`